src/components/emby-scroller/emby-scroller.css

```diff
@@ -25,4 +20,4 @@
         margin-left: 0;
         margin-right: 0;
     }
```

Comment:
```
.layout-tv .emby-scroller {
    padding-left: 3.3%;
    padding-right: 3.3%;
    margin-left: 0;
    margin-right: 0;
}
```
this will fix https://github.com/jellyfin/jellyfin-web/issues/569

Reply:
What if we just remove the media selector here? I forget why that was originally added, might have something to do with the mobile layout.

Reply:
yes indeed mobile layout used that too

Reply:
or you can change it to `.layout-mobile .emby-scroller`